src/extension.ts

```diff
@@ -10,6 +10,15 @@ import * as path from 'path';
 import * as Url from 'url';
 import * as fs from 'fs';
 
+function checkValidEditor() : boolean {
+    if (vscode.window.activeTextEditor === undefined) {
+        vscode.window.showErrorMessage('Document too large (or no editor selected). ' +
+            '5 MB limit on document size, see: https://github.com/Microsoft/vscode/issues/31078');
```

Comment:
Consider adding an FAQ in the README and link to it instead of the VS Code issue.  Have the FAQ link to the issue _and_ suggest the workaround that you mention in this changelog entry.